content/en/tracing/trace_explorer/trace_view.md

```diff
@@ -21,23 +21,86 @@ further_reading:
   text: "Understand how to read a Datadog Trace"
 ---
 
-View an individual [trace][1] to see all of its [spans][2] and associated metadata. Each trace can be viewed either as a flame graph or as a list (grouped by [service][3] or host).
+{{< img src="tracing/trace_view/trace_view.png" alt="Trace View" style="width:90%;">}}
 
-{{< img src="tracing/visualization/trace/trace.png" alt="Trace" style="width:90%;">}}
+## Overview
 
-Calculate the breakdown of execution time and adjust the color scheme by either **service** or **host**.
+View an individual [trace][1] to see all of its [spans][2] and associated metadata. Each trace can be visualized as either a Flame Graph, Span List, Waterfall, or Map.
 
-{{< img src="tracing/visualization/trace/service_host_display.png" alt="Service host display" style="width:40%;">}}
+The trace header displays critical trace information, including the root span's service name, resource name, trace ID, end-to-end trace duration, and the trace start datetime. To retain a trace indefinitely, click \"Open Full Page\" and save the URL.
```

Comment:
```suggestion
The trace header displays critical trace information, including the root span's service name, resource name, trace ID, end-to-end trace duration, and the trace start datetime. To retain a trace indefinitely, click **Open Full Page** and save the URL.
```
What do we mean by "retain a trace indefinitely"? I think this means if you want to quickly revisit a trace, save the URL somewhere. It immediately made me think of [trace retention](https://docs.datadoghq.com/tracing/trace_pipeline/trace_retention/), so I think we should use a different word if these aren't related concepts.

Reply:
These actually are related concepts - by clicking on **Open Full Page**, that specific trace is retained indefinitely instead of the default retention period (and can be accessed by the user at any time later via the URL)